### PR TITLE
Server, DB, SeedData

### PR DESCRIPTION
**Pull request by seth-way**
Created express server with api route.
Created MongoDB with schema.
Created SeedData.js.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  restaurant_id: Number,
+  restaurant_name: String,
+  menus: [{
+    menuName: String,
+    sections: [{
+      sectionName: String,
+      menuItems: [{
+        menuItemName: String,
+        description: String,
+        prices: [],
```
Comment (jsim0809) [inline database/index.js:25]: Double check here on this element of your schema. You are saying each menu item can have more than one price, right? Should this be: 
```suggestion
        prices: [Number],
```
?

Comment (jsim0809) [reply database/index.js:25]: Or maybe
```suggestion
        prices: [String],
```

File: database/scripts/dummy-data-helpers.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  return menuItems;
+};
+
+// method that takes a menu and generates an array of 2-7 sections
+// (some w/ a name string, some without)
+const generateSections = () => {
+  const numOfSections = Math.floor(Math.random() * 6) + 2;
+  const sections = [];
+  for (let i = 0; i < numOfSections; i += 1) {
+    const showSectionName = Math.random > 0.25;
```
Comment (jsim0809) [inline database/scripts/dummy-data-helpers.js:55]:
```suggestion
    const showSectionName = Math.random() > 0.25;
```

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const axios = require('axios');
+const db = require('../database/index.js');
+
+const app = express();
+const PORT = 3002;
+
+// TODO: update once static client files are available
+// app.use(express.static())
+app.use(express.json());
+app.use(express.urlencoded());
```
Comment (jsim0809) [inline server/index.js:12]: I think you might need to add this:
```suggestion
app.use(express.urlencoded({ extended: true }));
```
I'm not totally sure, but I think it allows you to parse nested objects (which you will be having a lot of).

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+app.get('/api/menu/:id', (req, res) => {
+  const { id } = req.params;
+  res.send(id);
```
Comment (jsim0809) [inline server/index.js:16]: Assuming this is test code for now. Maybe should have a todo flag?
```suggestion
  res.send(id); // TODO: query database and send back restaurant document
```

**Review by jsim0809 — COMMENTED** [review]
Went through and made various comments directly in-line -- take a look! Nice job writing that crazy menu-generating code.

**Comment by seth-way** [comment]
Closing pull request to make suggested changes. I will submit a new pull request once they are implemented.

**Pull request closed without merging**